src/main/java/org/folio/rest/impl/RequestPreferencesAPI.java

```diff
@@ -56,7 +59,7 @@ public void putRequestPreferenceStorageRequestPreferenceById(String id, String l
     return reply -> {
       if (isUniqueUserIdViolation(reply)) {
         asyncResultHandler.handle(
-          succeededFuture(Response.status(422).header("Content-Type", "application/json")
+          succeededFuture(Response.status(HttpStatus.HTTP_UNPROCESSABLE_ENTITY.toInt()).header("Content-Type", "application/json")
```

Comment:
Personally, I might statically import this constant to shorten this line. 

Please can it be split to conform with the maximum line length.

```suggestion
          succeededFuture(Response.status(HTTP_UNPROCESSABLE_ENTITY.toInt()).header("Content-Type", "application/json")
```